src/WarcraftLegacies.Source/Quests/Frostwolf/QuestRagetotem.cs

```diff
@@ -0,0 +1,34 @@
+﻿using MacroTools.QuestSystem;
+using static War3Api.Common;
+using MacroTools.ObjectiveSystem.Objectives.LegendBased;
+using MacroTools.LegendSystem;
+
+namespace WarcraftLegacies.Source.Quests.Frostwolf
+{
+  /// <summary>
+  /// Bring any hero to the Echo Isles to unlock the base.
+  /// </summary>
+  public sealed class QuestRagetotem : QuestData
+  {
+    private readonly LegendaryHero _cairne;
+
+    /// <inheritdoc />
+    public QuestRagetotem(LegendaryHero cairne) : base("Ragetotem Tribe Challenge",
+      "Reknown for their warriors, the Ragetotem Tribe has issued a challenge to Cairne. If he was to overcome it, they would join the Bloodhoof.",
```

Comment:
The objective doesn't really indicate that Cairne is responding to a challenge. He's just getting stronger.
```suggestion
      "The Ragetotem Tribe are renowned for their martial prowess. An older, mightier Cairne might convince them to join the Bloodhoof.",
```